dashboard/app/dsl/script_dsl.rb

```diff
@@ -45,6 +46,10 @@ def trophies(trophies_string)
     @trophies = ActiveRecord::ConnectionAdapters::Column::value_to_boolean(trophies_string)
   end
 
+  def wrapup_video(wrapup_video)
```

Comment:
Nice!  [We're already using `method_missing` in the External HTML DSL](https://github.com/code-dot-org/code-dot-org/pull/1271/files#diff-398f562d61134e1df17500acc2270254R10), so I think it makes a ton of sense to do something similar here.